server/server.js

```diff
@@ -5,6 +5,13 @@ const cors = require('cors');
 const db = require('../database/database.js');
 
 app.use(cors());
+
+app.get('*.js', function (req, res, next) {
```

Comment:
Can you explain what this get request is doing? Why are you passing in '*.js' as a parameter?

Reply:
Sure, this code allows the express server to serve the zipped (optimized) version of my bundle to the client.
This stackoverflow post goes into it a bit: https://stackoverflow.com/questions/49911058/cant-serve-gzipped-files-properly-using-express
I would consider looking into it! It turned some of my services from ratings of 70 to 100!